furryfeet_be/furryfeet/urls.py

```diff
@@ -10,4 +10,10 @@
     path('logout/', LogoutView.as_view()),
     path('user/', UserView.as_view()),
     path('user/<int:pk>/image/', AddUserImageView.as_view()),
+    path('dog-walker/', DogWalkerView.as_view(), name='dog-walker-get'),  # GET request to retrieve dog walker info, appointments and feedbacks
+    path('dog-walker/<int:pk>/', DogWalkerView.as_view(), name='dog-walker-update'),  # POST request to update dog walker info
+    path('appointments/', AppointmentView.as_view(), name='appointment-get'),  # GET request to retrieve appointments
+    path('appointments/<int:pk>/', AppointmentView.as_view(), name='appointment-create'),  # POST request to create a new appointment
+    path('feedbacks/', FeedbackView.as_view(), name='feedback-get'),  # GET request to retrieve feedbacks
+    path('feedbacks/<int:pk>/', FeedbackView.as_view(), name='feedback-create'),  # POST request to create a new feedback
 ]
```

Comment:
Here you dont need the POST request urls, i think django calls the view, checks the request type and based on it calls the corresponding function defined in the view. @StankovskiA correct me if im wrong here